### PR TITLE
feat: custom datetime format

### PR DESCRIPTION
**Pull request by dspo**
feat: custom datetime format

### CODE REVIEW

File: docs/1.6/manual/dop/guides/reference/pipeline.md [hunk 1]
```diff
@@ -752,7 +752,29 @@ timestamp_ns_hour：1小时前时间戳格式：1586913750801408626，单位是
...
+- ${{ random.datetime_custom_RFC850 }}: `Tuesday, 08-Mar-22 16:33:06 CST`
+- ${{ random.datetime_custom_RFC1123 }}: `Tue, 08 Mar 2022 16:33:06 CST`
+- ${{ random.datetime_custom_RFC1123Z }}: `Tue, 08 Mar 2022 16:33:06 +0800`
+- ${{ random.datetime_custom_RFC3339 }}: `2022-03-08T16:33:06+08:00`
+- ${{ random.datetime_custom_RFC3339Nano }}: `2022-03-08T16:33:06.629310528+08:00`
+- ${{ random.datetime_custom_Kitchen }}: `4:33PM`
+- ${{ random.datetime_custom_Stamp }}: `Mar  8 16:33:06`
+- ${{ random.datetime_custom_StampMilli }}: `Mar  8 16:33:06.629`
+- ${{ random.datetime_custom_StampMicro }}: `Mar  8 16:33:06.629326`
+- ${{ random.datetime_custom_StampNano }}: `Mar  8 16:33:06.629330204`
```
Comment (sfwn) [inline docs/1.6/manual/dop/guides/reference/pipeline.md:772]: please check it: `Mar  8` two space between `Mar` and `8` is right or not.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Comment by sfwn** [comment]
related to erda pr please.

**Review by sfwn — APPROVED** [review]

File: docs/1.6/manual/dop/guides/reference/pipeline.md (hunk 1 above)
Comment (dspo) [reply docs/1.6/manual/dop/guides/reference/pipeline.md:772]: yes

**Pull request merged**